packages/quantic/force-app/main/translations/fr.translation-meta.xml

```diff
@@ -484,4 +484,44 @@
       <label>Voir moins</label>
       <name>quantic_SmartSnippetShowLess</name>
     </customLabels>
+    <customLabels>
```

Comment:
@acarpentier01, here are the french translations.
any feedback would be welcome, thank you!